src/main/java/org/mule/extensions/jms/internal/connection/provider/activemq/ActiveMQConnectionProvider.java

```diff
@@ -35,10 +45,26 @@
  * @since 1.0
  */
 @Alias("active-mq")
-@ExternalLib(name = "ActiveMQ", description = "An ActiveMQ Broker", type = DEPENDENCY)
+@ExternalLib(name = "ActiveMQ Client", description = "The ActiveMQ JMS Client implementation.", type = DEPENDENCY,
+    requiredClassName = CONNECTION_FACTORY_CLASS, coordinates = "org.apache.activemq:activemq-client:" + ACTIVEMQ_VERSION)
+@ExternalLib(name = "ActiveMQ Broker",
+    description = "The ActiveMQ Message Broker implementation. Only required if it is required to start an on memory broker based in the VM transport.",
```

Comment:
"Only required when using an in-memory broker based on the VM transport, like the one configured by default."

Reply:
is there a way/need here to say that is one or the other?